src/molecules/gv-confirm.js

```diff
@@ -151,6 +160,31 @@ export class GvConfirm extends GvPopover {
     if (this.danger) {
       this.shadowRoot.querySelectorAll('gv-button').forEach((btn) => (btn.setAttribute('danger', '')));
     }
+    this._disabled = this._calculateIsDisabled();
+    if (this.disabled) {
+      this._reflectDisabledOnSlot();
+    }
+  }
+
+  _reflectDisabledOnSlot () {
+    const slot = this.shadowRoot.querySelector('slot');
+    const childNodes = slot.assignedNodes({ flatten: true });
+    childNodes.forEach((node) => {
+      node.disabled = true;
+      return node.disabled;
+    });
+  }
+
+  _calculateIsDisabled () {
+    const slot = this.shadowRoot.querySelector('slot');
+    const childNodes = slot.assignedNodes({ flatten: true });
+    let isSlotButtonDisabled = false;
+    childNodes.forEach((node) => {
+      if (node.nodeType === Node.ELEMENT_NODE) {
+        isSlotButtonDisabled = node.getAttribute('disabled');
+      }
+    });
```

Comment:
With this `forEach` we get the activation state of the last sub-node, I don't know if it is the intended behavior.
Here is a suggestion with `filter` + `some` to take into account all the sub node.
```suggestion
    const isSlotButtonDisabled = childNodes
      .filter((node) => node.nodeType === Node.ELEMENT_NODE)
      .some((node) => Boolean(node.getAttribute('disabled')) === true);
```

I also added a `Boolean(...)` cast as `node.getAttribute('disabled')` is returning a string, like `"true"`.

Reply:
:+1: